installer/install.bat

```diff
@@ -0,0 +1,94 @@
+@echo off
+
+@rem This script will install git and conda (if not found on the PATH variable)
+@rem  using micromamba (an 8mb static-linked single-file binary, conda replacement).
+@rem For users who already have git and conda, this step will be skipped.
+
+@rem Next, it'll checkout the project's git repo, if necessary.
+@rem Finally, it'll create the conda environment and preload the models.
+
+@rem This enables a user to install this project without manually installing conda and git.
+
+echo "Installing InvokeAI.."
+echo.
+
+@rem config
+set MAMBA_ROOT_PREFIX=%cd%\installer_files\mamba
+set INSTALL_ENV_DIR=%cd%\installer_files\env
+set MICROMAMBA_DOWNLOAD_URL=https://github.com/cmdr2/stable-diffusion-ui/releases/download/v1.1/micromamba.exe
```

Comment:
Self-contain to invokeai? When would we make this update?

Reply:
Probably after this is merged to dev we can open a separate PR to add it